Fastpack/Cache.rei

```diff
@@ -0,0 +1,27 @@
+type t
+type load =
+  | Empty
+  | Load(params)
+and params = {
+  currentDir: string,
+  projectRootDir: string,
+  mock: list((string, Config.Mock.t)),
+  nodeModulesPaths: list(string),
+  resolveExtension: list(string),
+  preprocess: list(Preprocessor.config),
+};
+
+let make: load => Lwt.t(t);
+let isLoadedEmpty: t => bool;
```

Comment:
Better name `isLoadedFresh`, I think it describes it more fully (cache can be loaded from disk but still be empty, right?).

Reply:
Well, probably even better name would be `isEmptyInitially`. We don't care how it was loaded, we care if it is initially empty. As for the "loading empty from disk" - this is possible theoretically, but very unlikely (I'd say impossible) practically given the workflow